Fix Javadoc warnings

### Describe the bug

There are javadoc warnings

### Expected behavior

_No response_

### Minimal reproducible example

_No response_

### Add-on Version

2.2.0-SNAPSHOT

### Vaadin Version

-

### Additional information

```
[WARNING] Javadoc Warnings
[WARNING] J:\fc\addons\GoogleMapsAddon\src\main\java\com\flowingcode\vaadin\addons\googlemaps\GoogleMap.java:72: warning: Tag @link: reference not found: GoogleMapStatestate
[WARNING] * Initiates a new GoogleMap object with default settings from the {@link GoogleMapState state
[WARNING] ^
[WARNING] object
[WARNING] J:\fc\addons\GoogleMapsAddon\src\main\java\com\flowingcode\vaadin\addons\googlemaps\GoogleMap.java:72: warning: Tag @link: reference not found: GoogleMapStatestate
[WARNING] * Initiates a new GoogleMap object with default settings from the {@link GoogleMapState state
[WARNING] ^
[WARNING] object
[WARNING] J:\fc\addons\GoogleMapsAddon\src\main\java\com\flowingcode\vaadin\addons\googlemaps\GoogleMap.java:437: warning: @param argument "url" is not a parameter name.
[WARNING] * @param url to be displayed.
[WARNING] ^
[WARNING] J:\fc\addons\GoogleMapsAddon\src\main\java\com\flowingcode\vaadin\addons\googlemaps\GoogleMap.java:72: warning: Tag @link: reference not found: GoogleMapStatestate
[WARNING] * Initiates a new GoogleMap object with default settings from the {@link GoogleMapState state
[WARNING] ^
[WARNING] object
[WARNING] 4 warnings
```

--- a/src/main/java/com/flowingcode/vaadin/addons/googlemaps/GoogleMap.java
+++ b/src/main/java/com/flowingcode/vaadin/addons/googlemaps/GoogleMap.java
@@ -69,8 +69,7 @@ public class GoogleMap extends Component implements HasSize {
   }
 
   /**
-   * Initiates a new GoogleMap object with default settings from the {@link GoogleMapState state
-   * object}.
+   * Initiates a new GoogleMap object.
    *
    * @param apiKey The Maps API key from Google. Not required when developing in localhost or when
    *     using a client id. Use null or empty string to disable.
@@ -434,7 +433,7 @@ public class GoogleMap extends Component implements HasSize {
   /**
    * Sets a KML or GeoRSS feed url to be displayed as a KML Layer in the map.
    * 
-   * @param url to be displayed.
+   * @param kml the url to be displayed.
    */
   public void setKml(String kml){
     this.getElement().setProperty("kml", kml);

